packages/extension/src/ui/action/views/swap/libs/send-transactions.ts

```
@@ -238,15 +238,15 @@ export const executeSwap = async (
                   // Just continue and hope for the best with old block hash...
                   console.warn(
                     `Failed to get latest blockhash after ${backoffi} attempts,` +
-                      ` continuing with old block hash for legacy transaction...`,
+                    ` continuing with old block hash for legacy transaction...`,
                   );
                   break update_block_hash;
```

Biome

[error] Unnecessary label.

Safe fix: Remove the unnecessary label.
You can achieve the same result without the label.


(lint/complexity/noUselessLabel)